src/core/operations_problem_results.jl

```diff
@@ -157,3 +159,13 @@ end
 function write_to_CSV(results::OperationsProblemResults, folder_path::String)
     write_results(results, folder_path; file_type = CSV)
 end
+
+function _find_params(variables::Array)
```

Comment:
I think this function is replicated from https://github.com/NREL/PowerSimulations.jl/blob/e2cd5f2df16799ed89496b4194cc9cead0716991/src/core/operations_problem_results.jl#L32